src/network/networkd-network-gperf.gperf

```diff
@@ -37,6 +37,7 @@ Link.Multicast,                         config_parse_tristate,
 Link.AllMulticast,                      config_parse_tristate,                          0,                             offsetof(Network, allmulticast)
 Link.Unmanaged,                         config_parse_bool,                              0,                             offsetof(Network, unmanaged)
 Link.RequiredForOnline,                 config_parse_bool,                              0,                             offsetof(Network, required_for_online)
+Link.RequiredOperationalStateForOnline, config_parse_link_operstate,                    0,                             offsetof(Network, required_operstate_for_online)
```

Comment:
maybe fold this into the existing RequiredForOnline=? i.e. parse the thing as both a boolean and as a specific state? I thinkt it would be nice to minimize the settings, and given that this is kinda the same thing this would make some sense to me?

i.e. RequiredForOnline=yes and RequiredForOnline=degraded would be the same

Reply:
I considered the possibility of that. But, I'd like to use the value specified by `RequiredOperationalStateForOnline=` even if `RequiredForOnline=no`. Because, `RequiredForOnline=no` can be override by `-i` option of `wait-online`. And if interfaces are specified in `-i` option without optional operational state, e.g., `-i eth0`, then the required operational state is retrieved from `RequiredOperationalStateForOnline=`.

So, if `eth0` matches a .network file with
```
RequiredForOnline=no
RequiredOperationalStateForOnline=carrier
```
then, 
- `wait-online` ignores `eth0`,
- `wait-online -i eth0` wait for that `eth0` gained a carrier, i.e. it is equivalent to `wait-online -i eth0:carrier`
- `wait-online -i eth0:routable` wait for that `eth0` is fully configured.

Also, in a later PR, I am planning to change the default value of `RequiredOperationalStateForOnline=` depending on the configuration: bonding or bridge slave, stacked netdevs, netdev kind, etc.

Reply:
Ah, after re-considering your comment, I noticed that the link state file should have two related variables, but for most cases, it is not necessary .network file to have the two settings. OK. I will merge the settings.